bin/http-server

```diff
@@ -7,9 +7,47 @@ var colors     = require('colors/safe'),
     httpServer = require('../lib/http-server'),
     portfinder = require('portfinder'),
     opener     = require('opener'),
-    argv       = require('optimist')
-      .boolean('cors')
-      .argv;
+    parseArgs  = require('minimist');
+
+var argv = parseArgs(process.argv.slice(2), {
+  alias: {
```

Comment:
Can we switch the aliases, so the variable names are clearer? `argv.silent` instead of `argv.s`?

Reply:
Sure! That's a good idea. I'll make those changes in a bit